crates/bevy_ui/src/experimental/ghost_hierarchy.rs

```diff
@@ -20,7 +21,8 @@ use crate::Node;
 #[derive(Component, Debug, Copy, Clone, Reflect)]
 #[cfg_attr(feature = "ghost_nodes", derive(Default))]
 #[reflect(Component, Debug)]
-#[require(Visibility, Transform)]
+#[cfg_attr(feature = "bevy_render", require(Visibility, Transform))]
+#[cfg_attr(not(feature = "bevy_render"), require(Transform))]
```

Comment:
Future PR: Might be nice to modify the `Component` derive macro to allow multiple `#[require(...)]` attributes (if that's possible?). Would make cases like this cleaner.

Reply:
I was mistaken, you can actually do this right now, I just assumed it didn't work because I've never seen it written. You should be able to replace this with:

```rust
#[require(Transform)]
#[cfg_attr(feature = "bevy_render", require(Visibility))]
```